returnn/torch/engine.py

```diff
@@ -3,15 +3,14 @@
 """
 
 from __future__ import annotations
-from typing import Optional, Any, Union, Callable, Dict
```

Comment:
But `Callable` is used many times here?

Reply:
i only saw it in comments

Reply:
See for example:
```
  returnn/torch/engine.py:74: WARNING PyUnresolvedReferencesInspection: Unresolved reference 'Callable'
  returnn/torch/engine.py:75: WARNING PyUnresolvedReferencesInspection: Unresolved reference 'Callable'
```

Reply:
> i only saw it in comments

Those are type information. It is used then. It must be imported.